Lib/test/test_enum.py

```diff
@@ -2893,6 +2893,14 @@ def __new__(cls, c):
         self.assertEqual(FlagFromChar.a, 158456325028528675187087900672)
         self.assertEqual(FlagFromChar.a|1, 158456325028528675187087900673)
 
+    def test_multiple_superclasses(self):
```

Comment:
We have tests for `repr` defined here: https://github.com/python/cpython/blob/ffa20484dfcd3e542d3fb76137160abc2a1f98a0/Lib/test/test_enum.py#L612

Let's move it closer maybe?